rules/0575-windows_rules.xml

```diff
@@ -0,0 +1,5187 @@
+    <description>Unable to terminate notifications normally</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20436" level="5">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7011$</field>
+    <description>Directory location invalid</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20437" level="5">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7013$</field>
+    <description>Disk full, update paused</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20438" level="5">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7040$</field>
+    <description>Index data cannot be read</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20439" level="5">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7066$</field>
+    <description>Missing directory configuration</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20440" level="5">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7068$</field>
+    <description>Cannot read registry, registry keys missing</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20441" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^1003$</field>
+    <description>The Windows Search Service started</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20442" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^1013$</field>
+    <description>The Windows Search Service stopped normally</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20443" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^7064$</field>
+    <description>Performance monitoring cannot be initialized because the counters are not loaded or the shared memory object cannot be opened</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20444" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^1010$</field>
+    <description>The Windows Search Service removed the old search index successfully</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20445" level="5">
+    <if_sid>20002</if_sid>
+    <field name="win.system.providerName">Microsoft-Windows-Search</field>
+    <field name="win.system.eventID">^1008$</field>
+    <description>The Windows Search Service attempted to remove the old search index</description>
+    <options>no_full_log</options>
+  </rule>  
+
+  <rule id="20447" level="0">
+    <if_sid>20001</if_sid>
+    <field name="win.system.providerName">VSS</field>
+    <description>Group of VSS events</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20448" level="3">
+    <if_sid>20447</if_sid>
+    <field name="win.system.eventID">^8224$</field>
+    <desc
```

Comment:
This rule is not valid. Windows 2000 doesn't support the eventchannel format. Please review if any other rules are related to this OS.